src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/reroute/actions/UpdateFlowStatusAction.java

```diff
@@ -48,6 +48,9 @@ protected void perform(State from, State to, Event event, FlowRerouteContext con
         FlowStatus resultStatus = persistenceManager.getTransactionManager().doInTransaction(() -> {
             Flow flow = getFlow(flowId, FetchStrategy.DIRECT_RELATIONS);
             FlowStatus flowStatus = flow.computeFlowStatus();
+            if (stateMachine.isIgnoreBandwidth() && flowStatus == FlowStatus.UP) {
+                flowStatus = FlowStatus.DEGRADED;
```

Comment:
In the current implementation, we have DEGRADED status only when we could not find the protected path. But in this case, this status is set for another reason. Therefore, the message `Couldn't find non overlapping protected path` in `status_info` will be incorrect.

Reply:
good catch, haven't seen this before rebase, fixed